CONTRIBUTING.md

```diff
@@ -112,12 +112,12 @@ For testing at development time, we make use of three tools:
    pytype weather_dl
    ```
 
-* `flake8` for linting:
+* `black` for linting:
```

Comment:
Let's change these to `pyink`.

Reply:
Thank you so much for reviewing my PR this much interestingly @alxmrs . In here I thought to activate pyink that there should be black. I think I did mistake. Yeah we can change that to `pyink` . I'm trying new approches to make another PR for this issue. Can you please read my new approches Alex?